tests/test_null.c

```diff
@@ -31,7 +30,7 @@ int main(void)
 		printf("parsed string:   ");
 		puts(json);
 		puts("FAIL");
-		retval=1;
+		return 1;
```

Comment:
the idea here was that if the test fails here, stop it here, so that we don't generate more output
but, since the output is compared against an expected output the test should fail;
will fix